requirements.txt

```diff
@@ -1,3 +1,3 @@
 intake
-geopandas
+-e git://github.com/geopandas/geopandas
```

Comment:
Usually I fix packages in an environment file... this here doesn’t work

Reply:
Tried https://stackoverflow.com/questions/16584552/how-to-state-in-requirements-txt-a-direct-github-source